docs/data/dataset_types.md

```diff
@@ -1,53 +1,82 @@
 ---
-sidebar_position: 3
+sidebar_position: 1
 ---
 
 # Dataset Types
 
-The datasets in GOAT are divided into **internal and external datasets**. Internal datasets are those that are stored within GOAT, while external datasets are those that are sourced from third-party services. 
+Users can access datasets in the **Catalog Explorer** and through the **Dataset Explorer**. The datasets found in the [Catalog](../workspace/Catalog) are provided and maintained by Plan4Better. If you add an external dataset via URL, upload a dataset from your local computer or create a layer in GOAT, they will be visible in the Dataset Explorer.
 
-## Internal Datasets
+**Adding Datasets**
 
-Internal datasets refer to datasets that are **stored and managed directly in GOAT**. These include [Features](#1-features), which contain spatially referenced geographic features, and [Tables](#2-tables), which consist of non-spatial data. These internal datasets are essential for performing analysis in GOAT.
+![GOAT data types](/img/data/data_basis/original_files/dataset_types.png "Dataset Types")
+
+
+:::info External Datasets
+
+Unlike other datasets, external datasets are sourced from **third-party services** via the link you provide. They will be fetched into GOAT and stored there. These datasets can either be [Features](#1-features) or [Rasters](#2-rasters), each serving distinct purposes. The following external datasets are supported in GOAT: Web Map Service (WMS), Web Map Tile Service (WMTS), Web Feature Service (WFS), XYZ Tiles.
```

Comment:
@NNoemi1 
External feature layers will be fetched and stored in GOAT, but external raster layers will be fetched live (to overlay on the map) and not stored. Could we clarify this in the paragraph?